content/en/basics/how-retrieval-works/basic-retrieval/index.md

```diff
@@ -127,11 +268,36 @@ Now that we’ve downloaded a CAR file, we need to find out what’s inside it.
     -rw-rw-r-- 1 user user 143M Feb 16 11:36 moon-data.tar.gz
     ```
 
-1. You can then manage the data as you need.
+4. You can then manage the data as you need.
 
 And there we have it! Downloading and managing data from Filecoin is super simple when you use Lassie and Go-car!
 
+### Lassie CLI/API
+
+Lassie’s API and CLI currently consist of a single command: `fetch`. Fetch takes a CID and outputs a CAR file. Eventually, we’d like the Lassie CLI and HTTP API to support the following features:
+
+1. Direct Provider Fetch
+
+    ```
+    lassie fetch CID --providers ....
+    ```
+
+2. Bitswap only retrieval
+
+    ```
+    lassie fetch CID --disable-graphsync
+    ```
```

Comment:
```suggestion
2. Bitswap only retrieval

    ```
    lassie fetch CID --disable-graphsync
    ```
```

Let's ditch this, it's advanced usage and as of a few days ago this option has been removed and replaced with a `--protocols` argument. I'm not sure that's useful for the audience though—mostly they just want to fetch their content however they can. `--providers` is even advanced but maybe not unreasonable if they know a specific SP they want to fetch from.